src/test/transaction_tests.cpp

```diff
@@ -38,6 +38,25 @@ extern UniValue read_json(const std::string &jsondata);
 
 BOOST_FIXTURE_TEST_SUITE(transaction_tests, BasicTestingSetup)
 
+BOOST_AUTO_TEST_CASE(tx_standard_creation)
+{
+    std::vector<unsigned char> vec(1);
```

Comment:
`vec` is unused

Reply:
nice catch!  its supposed to be script << vec not script << qty